device/cluster.cpp

```diff
@@ -1432,43 +1429,47 @@ void Cluster::init_pcie_iatus() {
 
 int Cluster::test_setup_interface() {
     int ret_val = 0;
-    int logical_device_id = m_pci_device_map.begin()->first;
-    PCIDevice* dev = m_pci_device_map.at(logical_device_id).get();
+    int logical_device_id = m_tt_device_map.begin()->first;
+    TTDevice* tt_device = m_tt_device_map.at(logical_device_id).get();
     if (arch_name == tt::ARCH::GRAYSKULL) {
-        uint32_t mapped_reg = dev->set_dynamic_tlb(
-                                     dev->get_architecture_implementation()->get_reg_tlb(),
-                                     tt_xy_pair(0, 0),
-                                     0xffb20108,
-                                     harvested_coord_translation.at(logical_device_id))
+        uint32_t mapped_reg = tt_device
+                                  ->set_dynamic_tlb(
+                                      tt_device->get_architecture_implementation()->get_reg_tlb(),
+                                      tt_xy_pair(0, 0),
+                                      0xffb20108,
+                                      harvested_coord_translation.at(logical_device_id))
                                   .bar_offset;
 
         uint32_t regval = 0;
-        dev->read_regs(mapped_reg, 1, &regval);
+        tt_device->read_regs(mapped_reg, 1, &regval);
         ret_val = (regval != 0xffffffff && ((regval & 0x1) == 1)) ? 0 : 1;
         return ret_val;
     } else if (arch_name == tt::ARCH::WORMHOLE_B0) {
-        uint32_t mapped_reg = dev->set_dynamic_tlb(
-                                     dev->get_architecture_implementation()->get_reg_tlb(),
-                                     tt_xy_pair(1, 0),
-                                     0xffb20108,
-                                     harvested_coord_translation.at(logical_device_id))
+        uint32_t mapped_reg = tt_device
+                                  ->set_dynamic_tlb(
+                                      tt_device->get_architecture_implementation()->get_reg_tlb(),
+                                      tt_xy_pair(1, 0),
+                                      0xffb20108,
+                                      harvested_coord_translation.at(logical_device_id))
                                   .bar_offset;
 
         uint32_t regval = 0;
-        dev->read_regs(mapped_reg, 1, &regval);
+        tt_device->read_regs(mapped_reg, 1, &regval);
         ret_val = (regval != 0xffffffff && (regval == 33)) ? 0 : 1;
         return ret_val;
     } else if (arch_name == tt::ARCH::BLACKHOLE) {
         // MT Inital BH - Try to enable this, but double check "regval == 33"
-        // int ret_val = 0;
-        // PCIDevice *dev = m_pci_device_map.begin()->second->hdev;
 
-        // uint32_t mapped_reg = dev->set_dynamic_tlb(m_pci_device_map.begin()->second,
-        // dev->get_architecture_implementation()->get_reg_tlb(), tt_xy_pair(1, 0), 0xffb20108,
-        // harvested_coord_translation.at(logical_device_id)).bar_offset;
+        // uint32_t mapped_reg = tt_device
```

Comment:
off topic for this PR, but do we know what is the exact issue with this commented code. I see Milos's comment above, but not really sure what is the problem just by reading it. Can we at least track this with the issue as it is

Reply:
https://github.com/tenstorrent/tt-umd/issues/367